### ISSUE TITLE
Update footer in app to match website

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#547]
### Problem you are trying to address with this feature
As discussed in [website backlog](https://trello.com/c/WrI8ApLc), the styling of activity pages is not consistent with the Hypothesis website. Nate has captured the main points in this screenshot:
![activity_page_footer](https://user-images.githubusercontent.com/12087705/37612290-d168b25e-2b72-11e8-9d0f-d5aa3a5b5af8.png)

**Please note that the summary below is just a writeup of what Nate has suggested. These would need to be reviewed/approved by Product/Design before action can be taken**

### Your solution
The main tasks seem to be:
- [ ] Color scheme: Change background to #DDDDDD, logo to black version, make text #7a7a7a
- [ ] Branding: Change "Hypothes.is" to "Hypothesis"
- [ ] Links: Logo and wordmark should link to web.hypothes.is instead of hypothes.is. For the rest of the links, instead of relying on redirects, change all to web.hypothes.is. 
- [ ] Nav: left links reflect top nav of website, right-most links reflect footer of website.

**Comment by dawachan** [comment]
Good catch @xolotl and @klemay. Yes definitely important to bring the activity page footer into alignment. Once you've done the changes before going live.. am happy to review if need be - just @mention me! cheers
